### DISCUSSION TITLE
406 error running on Heroku

### DISCUSSION THREAD

**Discussion by SzuJanLin** [kinngh/shopify-node-express-mongodb-app#74] [Q&A] [answered]
I got an 406 error after trying to install the app 

 at=info method=GET path="/auth/tokens?code=5d4fdc2e2e43f169cd1a5069a71c82c9&hmac=3333a30af0bce357629b901c331702110f34e7b4d33cd24cfbf1cf6f3412deba&host=YWRtaW4uc2hvcGlmeS5jb20vc3RvcmUvbGl2ZS1mbGFyZQ&shop=live-flare.myshopify.com&state=678240428241096&timestamp=1680733332" host=flareai.herokuapp.com request_id=60a4a265-2301-4d99-bb7f-46e760bbcdc4 fwd="66.219.195.186" dyno=web.1 connect=0ms service=352ms status=500 bytes=523 protocol=https
 ---> Error at /auth/tokens HttpResponseError: Received an error response (406 Not Acceptable) from Shopify:
 If you report this error, please include this id: fdbea074-1cd9-4402-aaf3-f3d5722d3e70
 at NewHttpClient.throwFailedRequest (/app/node_modules/@shopify/shopify-api/lib/clients/http_client/http_client.js:193:23)
 at NewHttpClient.<anonymous> (/app/node_modules/@shopify/shopify-api/lib/clients/http_client/http_client.js:220:22)
 at Generator.next (<anonymous>)
 at fulfilled (/app/node_modules/tslib/tslib.js:164:62)
 at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
 response: {
 code: 406,
 statusText: 'Not Acceptable',
 body: {},
 headers: {
 'Alt-Svc': [Array],
 'Cf-Cache-Status': [Array],
 'Cf-Ray': [Array],
 Connection: [Array],
 'Content-Security-Policy': [Array],
 'Content-Type': [Array],
 Date: [Array],
 Http_x_shopify_shop_api_call_limit: [Array],
 Nel: [Array],
 'Referrer-Policy': [Array],
 'Report-To': [Array],
 Server: [Array],
 'Server-Timing': [Array],
 'Strict-Transport-Security': [Array],
 'Transfer-Encoding': [Array],
 'X-Content-Type-Options': [Array],
 'X-Dc': [Array],
 'X-Download-Options': [Array],
 'X-Frame-Options': [Array],
 'X-Permitted-Cross-Domain-Policies': [Array],
 'X-Request-Id': [Array],
 'X-Shardid': [Array],
 'X-Shopid': [Array],
 'X-Shopify-Api-Version': [Array],
 'X-Shopify-Shop-Api-Call-Limit': [Array],
 'X-Shopify-Stage': [Array],
 'X-Sorting-Hat-Podid': [Array],
 'X-Sorting-Hat-Shopid': [Array],
 'X-Stats-Apiclientid': [Array],
 'X-Stats-Apipermissionid': [Array],
 'X-Stats-Userid': [Array],
 'X-Xss-Protection': [Array]
 }
 }
 }

<img width="1009" alt="Screen Shot 2023-04-05 at 4 52 44 PM" src="https://user-images.githubusercontent.com/55773421/230229296-593ed604-0877-4aea-9e58-ba955cde6e7b.png">

Anyone could shed some lights on this issue?
I have everything updated in authRedirect.js and auth.js

**Comment by SzuJanLin** [comment, marked as answer]
Nvm i figured out the issue, my SHOPIFY_API_VERSION="2022-10" should be SHOPIFY_API_VERSION=2022-10

I'm leaving this just for future reference if someone had the same error

**Reply by kinngh** [reply]
That's a known issue on Heroku (and almost every other deployment platform)!
